packages/medusa/src/services/event-bus.ts

```diff
@@ -37,7 +53,7 @@ export default class EventBusService {
   protected readonly logger_: Logger
   protected readonly stagedJobRepository_: typeof StagedJobRepository
   protected readonly jobSchedulerService_: JobSchedulerService
-  protected readonly observers_: Map<string | symbol, Subscriber[]>
+  protected readonly observers_: Map<string | symbol, Observer[]>
```

Comment:
todo: depends on the previous comment, would become `observersDescriptors_` to avoid any confusion between subscriber, observer etc